### ISSUE TITLE
Keyboard shortcut worst experience with the new version

### ISSUE THREAD

**Issue by Sov3rain** [xanderfrangos/twinkle-tray#843]
Hey there,

Updated the app the to latest version (1.16.0) and I miss a feature that was previously present:
When adjusting the brightness up or down, it used to show the new value in the overlay. Now it doesn't.

I'm using the app on Windows 10, fresh install, with all values to defaults. I've got 2 monitors (HP 27w).

Is it a known regression?

**Comment by xanderfrangos** [comment]
Yep, looks like there was a regression with the hotkey overlay. Could you let me know if it's correct in this build?: https://github.com/xanderfrangos/twinkle-tray/actions/runs/11152511468/artifacts/2008384089

**Comment by Sov3rain** [comment]
Hey @xanderfrangos, I tested v1.16.1 this morning. Things are a little bit better. Now the overlay appears most of the time, but with a noticeable delay (approx 1.5sec). Plus the UI is really broken on Win10: 
![image](https://github.com/user-attachments/assets/bfe43524-340f-4c9f-ad7d-38ccee13e0dc)

**Comment by xanderfrangos** [comment]
I believe I've solved the issue. Can you give this build a try?

https://github.com/xanderfrangos/twinkle-tray/actions/runs/11203408539/artifacts/2020962451

**Comment by snowtyler** [comment]
> I believe I've solved the issue. Can you give this build a try?
> 
> https://github.com/xanderfrangos/twinkle-tray/actions/runs/11203408539/artifacts/2020962451

Way better now. Thanks!

**Comment by Sov3rain** [comment]
@xanderfrangos Perfect, seems to be fixed. Everything looking fine (even the UI). I've got a last remark but don't know if it's related to Twinkle Tray or Windows: the tray icon does not save it's position. I placed it manually to be next to the volume tray icon, but it gets back in the "drawer" each time I restart my computer. Any idea? It appends only with the Twinkle Tray icon.
